bot/modules/orders/scenes.ts

```
@@ -64,8 +66,11 @@ export const createOrder = new Scenes.WizardScene(
       if (undefined === priceMargin && sats === 0)
         return createOrderSteps.priceMargin(ctx);
       if (undefined === method) return createOrderSteps.method(ctx);
-      // We remove all special characters from the payment method
-      const paymentMethod = method.replace(/[&/\\#,+~%.'":*?<>{}]/g, '');
+
+      let replaceRegex = /[&/\\#,+~%.'":*?<>{}]/g;
```

ESLint

[error] 'replaceRegex' is never reassigned. Use 'const' instead.

(prefer-const)